source/openapi-admin-v3.yaml

```diff
@@ -5525,7 +5547,71 @@ components:
         error_code:
           type: string
           description: The error type.
-    AtlasCluster:
+    AtlasConfigResponse:
```

Comment:
Can we still use `AtlasCluster` here? It's a bit more descriptive (this doesn't represent Atlas Federated Instances for example).